src/c-opentimelineio/tests/OTIOComposableTests.cpp

```diff
@@ -86,4 +57,21 @@ TEST_F(OpenTimelineIOTests, ComposableTest)
     AnyDictionaryIterator_destroy(itMetadata);
     AnyDictionaryIterator_destroy(itMetadataEnd);
     AnyDictionaryIterator_destroy(itMetadataResult);
+}
+
+TEST_F(OTIOCOmposableTests, SerializeTest)
+{
+    Any*                   value    = create_safely_typed_any_string("bar");
+    AnyDictionary*         metadata = AnyDictionary_create();
+    AnyDictionaryIterator* it = AnyDictionary_insert(metadata, "foo", value);
+    Composable*            seqi =
+        Composable_create_with_name_and_metadata("test", metadata);
+    Any* seqi_any =
+        create_safely_typed_any_serializable_object((SerializableObject*) seqi);
+    OTIOErrorStatus* errorStatus = OTIOErrorStatus_create();
+    const char* encoded = serialize_json_to_string(seqi_any, errorStatus, 4);
```

Comment:
I don't know why but serialization doesn't work now. The casting to Any* doesn't give any errors but the error status I get here is: `type mismatch while decoding`. I'll try looking into this again tomorrow. Meanwhile, does anyone have any idea why this might be happening?